package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "back-end-project-week",
+  "version": "1.0.0",
+  "description": "This week you will build a backend for a note taking app called \"Lambda Notes\".",
+  "main": "server.js",
+  "scripts": {
+    "test": "nodemon server.js"
```

Comment:
Generally the script set up to run the server is "start"  and test is used to run specific test files.   Ultimately you can set the script to any name you want.  But if this is the command to start your server, start would be more readable.